ru/index.md

```diff
@@ -1,24 +1,32 @@
 ---
 title: Elixir School
 layout: home
-version: 1.0.0
+version: 2.0.0
 ---
 
-[![License](//img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
-
-Базовые знания о языке Elixir, навеяно [Scala School](http://twitter.github.io/scala_school/) от Twitter.
-
-_Правки и сообщения о ошибках приветствуются!_
-
-## About Elixir
-
-"Elixir - динамический, функциональный язык программирования, разработанный для создания масштабируемых и легко поддерживаемых систем." — [elixir-lang.org](http://elixir-lang.org/)
-
-Elixir основан на проверенной временем виртуальной машине Erlang VM (BEAM) для разработки распределенных устойчивых к ошибкам систем с низким временем ответа.
-
-__Особенности__:
-
-+ Масштабируем
-+ Устойчив к ошибкам
-+ Функциональное программирование
-+ Расширяем
+Elixir School - первый ресурс для людей, которые хотят изучить язык программирования Elixir.
+
+Независимо от того, опытный ли вы разработчик или нет, вы найдёте то что нужно в уроках и вспомогательных ресурсах.
+
+Благодаря тяжёлому труду добровольцев, Elixir School был переведён на множество языков. Некоторые из этих переводов: [Việt ngữ][vi], [简体中文][zh-hans], [繁體中文][zh-hant], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][ms], [Українською][uk], [한국어][ko], [Italiano][it], [Deutsch][de], [বাংলা][bn], [Türkçe][tr], and [ภาษาไทย][th].
```

Comment:
I am not really sure whether it's [right](http://new.gramota.ru/spravka/punctum/punctum-attach1) but for me the first sentence reads more natural without a comma.